assemblies/assembly_patching-and-upgrading/main.adoc

```diff
@@ -37,10 +37,10 @@ include::{enterprise-dir}/installation/patches-applying-proc.adoc[leveloffset=+1
 == Additional resources
 * {URL_PLANNING_INSTALL}[_{PLANNING_INSTALL}_]
 ifdef::PAM[]
-* https://access.redhat.com/documentation/en-us/red_hat_process_automation_manager/7.1/html-single/patching_and_upgrading_red_hat_process_automation_manager_7.1/[_Patching and upgrading {PRODUCT} 7.1_]
+* https://access.redhat.com/documentation/en-us/red_hat_process_automation_manager/7.2/html-single/patching_and_upgrading_red_hat_process_automation_manager_7.2/[Patching and upgrading {PRODUCT} 7.2]
```

Comment:
@csherrar You left out the ital. Should be "[_Patching and ... 7.2_]" for both the PAM and DM links.

Reply:
Corrected.